linera-web/src/lib.rs

```diff
@@ -321,6 +321,19 @@ impl Client {
         Ok(())
     }
 
+    /// Gets the balance of the default chain.
+    ///
+    /// # Errors
+    /// If the chain couldn't be established.
+    pub async fn balance(&self) -> JsResult<String> {
```

Comment:
It should be fairly easy to get the balance of any chain I think 🤔 Actually, since all chains have the same state encoding for the native balance we should be able to query it w/o even downloading that chain